cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1459,7 +1459,7 @@ def test_json_responses(self):
         self.validate_course_xblock_info(json_response, course_outline=True)
 
     @ddt.data(
-        (ModuleStoreEnum.Type.split, 5, 5),
+        (ModuleStoreEnum.Type.split, 4, 4),
```

Comment:
@mattdrayer I could not understand this logic in case of split module. why number of calls are decreasing .....